account_move_line_sale_info/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 ForgeFlow S.L.
```

Comment:
```suggestion
# Copyright 2023 ForgeFlow S.L.
```
Just a minor comment

Reply:
@AlexPForgeFlow  you can use also # Copyright 2020-23 ForgeFlow S.L.